lib/presentation/redux/states/shared_space_details_state.dart

```diff
@@ -81,4 +81,13 @@ class SharedSpaceDetailsState extends LinShareState with EquatableMixin {
   SharedSpaceDetailsState setAccountQuota({Either<Failure, Success> viewState, AccountQuota newQuota}) {
     return SharedSpaceDetailsState(viewState, sharedSpace, membersList, activitiesList, newQuota);
   }
+
+  SharedSpaceDetailsState updateSharedSpaceMembers({Either<Failure, Success> viewState, List<SharedSpaceMember> oldMembers, SharedSpaceMember updateMember}) {
+    if (updateMember != null) {
+        var index = membersList.indexWhere((member) => member.account.accountId.uuid == updateMember.account.accountId.uuid);
```

Comment:
U should use `final` instead of `var` in this case. If the index does not need to change the value

Reply:
please remove if it does not need

Reply:
it still here.